libretro_core_options.h

```diff
@@ -1584,7 +1584,7 @@ struct retro_core_option_v2_definition option_defs_us[] = {
       BEETLE_OPT(cd_access_method),
       "CD Access Method (Restart Required)",
       NULL,
-      "Select method used to read data from content disk images. 'Synchronous' mimics original hardware. 'Asynchronous' can reduce stuttering on devices with slow storage. 'Pre-Cache' loads the entire disk image into memory when launching content, which may improve in-game loading times at the cost of an initial delay at startup. 'Pre-Cache' may cause issues on systems with low RAM, and will fall back to 'Synchronous' for physical media",
+      "Choose method used to read data from content disk images. 'Synchronous' mimics original hardware. 'Asynchronous' can reduce stuttering on devices with slow storage. 'Pre-Cache' loads the entire disk image into memory when launching content, which may improve in-game loading times at the cost of an initial delay at startup. 'Pre-Cache' may cause issues on systems with low RAM, and will fall back to 'Synchronous' for physical media.",
```

Comment:
"Choose the method", "mimics the original hardware", "slow storage" is self-explanatory enough? "when launching a content".

Reply:
1.: yes
2.: also, yes
3.: what do you mean? "on devices with slow storage" -> "on slow storage" ?
4.: absolutely not, this is the general case, no article required.

Reply:
For the third situation, perhaps I don't have a use case with slow storage (I do have a PSVita, but I never use it with RA). I think this is another case of me thinking in Spanish, as what I am missing is an extra word after "storage", not that are superfluous words.

Reply:
Oh, something like "on devices with a slow storage medium"? But I feel like storage is enough on its own in this case (for English).